http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtHttpClientBase.java

```diff
@@ -73,6 +75,11 @@ abstract class AwsCrtHttpClientBase implements SdkAutoCloseable {
                                                + "NettyNioAsyncHttpClient instead.");
         }
 
+        if (config.get(PROTOCOL_NEGOTIATION) == ProtocolNegotiation.ALPN) {
```

Comment:
Do we need to add validation here? CRT will only support ALPN for h2 (once we add h2 support)

Reply:
Removed validation for CRT